### ISSUE TITLE
NullPointerException Server Crash Cannot invoke "java.util.function.Supplier.get()" because "fruitLeaves" is null

### ISSUE THREAD

**Issue by natebc** [FTBTeam/FTB-Modpack-Issues#4086]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.2.0

### Log Files

TLDR;
```
net.minecraft.ReportedException: Exception ticking world
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:897) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:283) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.function.Supplier.get()" because "fruitLeaves" is null
	at net.tropicraft.core.common.block.TropicraftTrees.lambda$createFruit$3(TropicraftTrees.java:96) ~[Tropicraft-9.6.2.jar%23635!/:9.6.2]
	at net.tropicraft.core.common.block.TropicraftTrees$1.m_213817_(TropicraftTrees.java:134) ~[Tropicraft-9.6.2.jar%23635!/:9.6.2]
	at net.minecraft.world.level.block.SaplingBlock.m_222000_(SaplingBlock.java:46) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.world.level.block.SaplingBlock.m_213898_(SaplingBlock.java:37) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.world.level.block.state.BlockBehaviour$BlockStateBase.m_222972_(MixinBlockStateBase.java:762) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.level.ServerLevel.redirect$zpa000$redirectBlockStateTick(ServerLevel.java:5159) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.level.ServerLevel.m_8714_(ServerLevel.java:481) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.level.ServerChunkCache.m_8490_(ServerChunkCache.java:366) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.level.ServerChunkCache.m_201698_(ServerChunkCache.java:316) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:305) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893) ~[server-1.20.1-20230612.114412-srg.jar%23653!/:?]
```

logs/latest.log and crash-report covering the crash:
https://gist.github.com/natebc/d733833096e32ac4e670128dfd3a9bc3

### Describe the bug

Purchased all the Tropiccraft Saplings last night and planted them before logging out for the evening. This morning shortly after I logged in the server crash with the linked logs.

### Steps to reproduce

1. plant tropiccraft saplings.
2. when they grow crashy :(

### Expected behaviour

be able to enjoy some sweet pineapples

### Screenshots

N/A

### Additional information

Should be enough in the crash-report and logs I hope. If you need anything else, or would like me to try and reproduce in SP i can do that.

**Comment by natebc** [comment]
I went ahead and recreated in a SP world. Tropicraft trees don't respond to bone meal so i had to wait a little while for the growth tick to happen.

Here's a crash-report from that SP world.
https://gist.github.com/natebc/fcf20fc5ce5b42cf887caed7f2cd2e1d

**Comment by MvanRietschoten** [comment]
We basically got the same issue on version 1.1.0 of this DW20 modpack. When we started exploring Tropicraft. It happened right after this crashlog:
https://gist.github.com/MvanRietschoten/1e521cfacd3c195b12977a561480ef49

I know it is a hanging tick of 60 seconds, so maybe it is unrelated. In that case I just wanted to share that our server keeps crashing randomly when FTB Backup is running. No crashlog as a result and the world is put 5 min back in time.

**Comment by natebc** [comment]
Looks like a mod issue on tropicraft.

**Comment by NotActuallyTerry** [comment]
We have released v9.6.3 with a fix for this issue
